.pre-commit-config.yaml

```diff
@@ -10,6 +10,10 @@ repos:
     hooks:
       - id: identity
       - id: check-hooks-apply
+  - repo: https://github.com/psf/black-pre-commit-mirror
+    rev: 24.8.0
+    hooks:
+      - id: black
```

Comment:
I saw that .ipynb files are also formatted. Is the `black` hook capable of formatting jupyter notebook files? I've also noticed that there's a [`black-jupyter` hook](https://black.readthedocs.io/en/stable/integrations/source_version_control.html#jupyter-notebooks) for this purpose.

Reply:
Updated now thanks !!

I did `pip install "black[jupyter]"` then ran black locally before testing with the hooks.

https://black.readthedocs.io/en/stable/getting_started.html#installation